adserver/templates/adserver/includes/publisher-paid-notice.html

```diff
@@ -2,5 +2,6 @@
 
 
 {% if publisher and not publisher.allow_paid_campaigns %}
-<div class="alert alert-warning" role="alert">{% trans 'Your publisher account is not approved for paid campaigns. When you are ready for us to review your integration, please contact us.' %}</div>
+{% url 'support' as support_url %}
```

Comment:
I was noticing that this warning wasn't included on all pages. Should it be part of the base template instead of an include?

Reply:
Sure, that works.